### ISSUE TITLE
Qt wallet is giving aggregated balance 

### ISSUE THREAD

**Issue by IsmaelAznarCarmona** [VeriBlock/vbk-ri-btc#633]
Qt is giving aggregated balance from all the imported addresses, but we are not able to see which addresses that balance comes from through neither UI nor CLI

**Comment by Leonardiae** [comment]
Go to 'Settings', tab 'Wallet' and make sure 'Enable coin controle features' is enabled.
Then go to 'Send' and click 'Inputs'. Make sure 'Tree mode' is selected.
I think this makes visible where the aggregated balance is based off. 

One can also use the command listaddressgroupings

**Comment by Warchant** [comment]
I believe those coins that you think are "out of thin air" is just part of genesis coins. Will check shortly.

**Comment by Warchant** [comment]
`listreceivedbyaddress` returned 10% of my coins, but `listunspent` returned all coins. 

That is because `listreceivedbyaddress` by default skips coinbase transactions:
![image](https://user-images.githubusercontent.com/1867551/147224939-72b3458d-0060-447f-8c7b-a2c94c7610ce.png)

TLDR: QT is correct about showing you your total balance.

**Comment by IsmaelAznarCarmona** [comment]
With tree mode we where able to look at all the addresses and see the correct total balance.
